state.json

```diff
@@ -234,6 +234,8 @@
                 "random": true,
                 "commands": [
                     "gslide id=1gDlOjkeMb9xLHzxhb5P5XL7gpIq08w0hdjCdykRIGTY comment=\"Pocket slideshow\""
+                    "https://darksky.net/widget/default/37.779,-122.4199/us12/en.js?width=100%&height=600&title=San Francisco, CA&textColor=333333&bgColor=FFFFFF&transparency=false&skyColor=undefined&fontFamily=Arial&customFont=&units=us&htColor=333333&ltColor=C7C7C7&displaySum=yes&displayHeader=yes"
```

Comment:
I'm not sure what's going on, but this link has some extra code on the page that looks wrong. Maybe a line more like the Mountain View one would work better?

https://github.com/mozilla/moz-corsica/blob/ac098252b82a9636d7bc84933969539e7367be95/state.json#L94

Reply:
I'll try that, the widgets are a little different, so I wanted to see if darksky one would work